docs/import-cli.rst

```diff
@@ -260,6 +264,59 @@ Note: Connecting over SSH is necessary only if you intend to import in-place. If
+
+The following example shows how to do the import on CLI and follow-up operations like rendering and metadata import in one step.
+
+See (reference) for rendering and (reference) for metadata.
+
+In many cases, the rendering and metadata import is preferable to be done separately, as the visual checking of the imported images might be crucial for further rendering and metadata import, see :doc:`render` and :doc:`metadata` for details on this. 
+
+Further, the images you are importing might need a range or different rendeing settings, not just one settings for all of them. Also for this case, the step-by-step approach, first importing the images, then only deciding on the rendering strategy and preparing the `renderingdef.yml` files, is preferable.
+
+Nevertheless, there are cases which do not need visual checks and use a single rendering for all images, for which a streamlined sequence of commands is offered below which will perform all three steps (import, rendering and metadata import) in one single session on the CLI.
+
+Resources
+---------
+
+Additionally to the :ref:`Resources mentioned in the import-cli section<Resources>` and in the :ref:`Setup<Setup>` you will also need the rendering and metadata plugins as mentioned in :doc:`render` and :doc:`metadata`, and possibly the following files:
+
+   - https://github.com/ome/training-scripts/blob/master/maintenance/preparation/renderingdef.yml
+   - :download:`simple-annotation.csv <../scripts/simple-annotation.csv>`
+   - :download:`simple-annotation-bulkmap-config.yml <../scripts/simple-annotation-bulkmap-config.yml>`
+
+Step-by-step
+------------
+
+#. If you did not do so already, open a terminal window on your local machine and activate the conda environment where your ``omero-py`` is installed (see the :ref:`Setup<Setup>`)::
+
+   $ conda activate myenv
+
+#. Set the ``OMERODIR`` variable to point to the downloaded and unzipped OMERO server dir (see the :ref:`Setup<Setup>`)::
+
+   $ export OMERODIR=/path/to/OMERO.server-x.x.x-ice36-bxx
+
+#. Prepare an `renderingdef.yml` file, either creating a new one or downloading https://github.com/ome/training-scripts/blob/master/maintenance/preparation/renderingdef.yml.
+
+#. Prepare an `annotation.csv` file, by creating a new file or downloading the provided example file. In the example below, we use the file :download:`simple-annotation.csv <../scripts/simple-annotation.csv>`. The Dataset  names in this `CSV` file must match the Dataset names in OMERO as created and stored in the `DID` variable defined in the command here below. The Image names in the `CSV` file must match the file names in your imported folder.
+
+#. Prepare a `bulkmap-config.yml` file. In the example below, we use the file :download:`simple-annotation-bulkmap-config.yml <../scripts/simple-annotation-bulkmap-config.yml>`.
+
+#. Login to the OMERO.server you wish to import to. This can be a remote server if you do not wish to import `in-place`.
```

Comment:
```suggestion
#. Log in to the OMERO.server you wish to import to. This can be a remote server if you do not wish to import `in-place`.
```
(generally with login and setup, the verb forms get the spaces)